test/api/CLI.test.js

```diff
@@ -1703,11 +1703,9 @@ describe("CLI API", () => {
       consoleSpy = jest.spyOn(global.console, "log");
       exitSpy = jest.spyOn(process, "exit").mockImplementation(() => {});
 
-      cli.program.option("--color [value]", "any color", "blue");
```

Comment:
Adding a duplicate option now throws an error, hence updating the test case to use an existing option.

![Uploading Screenshot 2024-11-04 at 9.47.34 AM.png…]()